src/styles/blocks/container.scss

```diff
@@ -0,0 +1,24 @@
+.container {
+  margin: 0 auto;
+  width: $container-width;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 50px 0;
+
+  &__card {
+    margin-right: 48px;
+  }
+
+  & > :nth-child(4n) {
+    margin-right: $reset;
+  }
+
+  & > :nth-last-child(-n+4) {
+    margin-top: 46px;
```

Comment:
The selector :nth-last-child(-n+4) might not behave as expected as it will apply the styles to the last 4 elements, regardless of their type. Make sure that you have less than 4 different types of elements inside .container or specify the element type.